lib/data_integrity_utils.pm

```diff
@@ -32,9 +32,21 @@ sub verify_checksum {
         my $checksum = get_required_var $image;
         $image =~ s/CHECKSUM_//;
         my $image_path = get_required_var $image;
+        my $dir_path;
+        $dir_path = '/var/lib/libvirt/images/' if check_var('BACKEND', 'svirt');
+        $dir_path = get_var('HYPERV_DISK', 'D:') . '\\cache\\' if check_var('VIRSH_VMM_FAMILY', 'hyperv');
         $image_path = $dir_path . basename($image_path) if $dir_path;
-        my $digest = digest_file_hex($image_path, "SHA-256");
-        record_info("$image Ok", "$image_path: Ok") && next if $checksum eq $digest;
+        my $digest;
```

Comment:
Same thing for this block, create a function that return the digest and encapsulate from line 39-48 or (I like more this option): `my $digest = check_var('BACKEND', 'svirt')) ? get_digest_svirt : digest_file_hex($image_path, "SHA-256");

Reply:
you could not find `digest_file_hex` in the worker for svirt? We added here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/cpanfile#L16

Reply:
For svirt `digest_file_hex` checks the image on worker, but the image needs to be checked on the host as on svirt worker is not a host.